northstar/Cargo.toml

```diff
@@ -92,3 +92,6 @@ runtime = [
 
 [build-dependencies]
 vergen = "3.1.0"
+tempfile = "3.2.0"
+npk = { path = "../npk" }
+tokio = { version = "1.1.1", features = ["rt"], optional = true }
```

Comment:
Does this need `rt`?
What does the `optional` flag do in `build-dependencies`?

Reply:
what I need is is a runtime to execute a task..is there a less intrusive way?
 the optional is wrong